src/components/Request/FormFields/Methods.tsx

```diff
@@ -0,0 +1,114 @@
+import { Label, Checkbox, Text } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import "components/Request/Request.css";
+import { FormField } from "components/Request/NewRequestForm";
+import { ClipboardCheckmarkRegular } from "@fluentui/react-icons";
+
+const Methods = ({ name, form }: FormField) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        htmlFor={name + "Id"}
+        size="small"
+        weight="semibold"
+        className="requestFieldLabel"
+        required
+      >
+        <ClipboardCheckmarkRegular className="requestFieldIcon" />
+        Announcement Method(s)
+      </Label>
+      <Controller
+        name="methods"
+        control={form.control}
+        rules={{
+          validate: (_value, formValues) => formValues.methods.length > 0,
+        }}
+        render={({ field }) => (
+          <>
+            <Checkbox
```

Comment:
Some options to consider:
1) Make this a constant and do a map to dynamically create checkboxes so it can be easy to add/remove entries and simplifies code
2) Turn it into a Multi-select dropdown control instead of individual checkbox controls